src/test/java/seedu/travel/storage/JsonSerializableTravelBuddyTest.java

```diff
@@ -28,9 +28,11 @@
     public void toModelType_typicalPlacesFile_success() throws Exception {
         JsonSerializableTravelBuddy dataFromFile = JsonUtil.readJsonFile(TYPICAL_PLACES_FILE,
                 JsonSerializableTravelBuddy.class).get();
+
         TravelBuddy travelBuddyFromFile = dataFromFile.toModelType();
         TravelBuddy typicalPlacesTravelBuddy = TypicalPlaces.getTypicalTravelBuddy();
-        assertEquals(travelBuddyFromFile, typicalPlacesTravelBuddy);
+
+        assertEquals(travelBuddyFromFile.toString(), typicalPlacesTravelBuddy.toString());
```

Comment:
I think this is not necessary? Can just use the non-toString version if results are the same, since the toString invokes an unnecessary function call

Reply:
I tried using without toString() but it throws an error which does not make sense.
![image](https://user-images.githubusercontent.com/35837791/54875838-eb0ecd00-4e40-11e9-9f1d-056401f33683.png)

Reply:
I got that error too, that's due to the TypicalPlaces.java places not matching those in typicalPlacesTravelBuddy.json file

Reply:
Thanks for the solution. I have implemented it and it works now.